src/order/galois_connection.lean

```diff
@@ -67,6 +67,11 @@ lemma monotone_intro (hu : monotone u) (hl : monotone l)
 
 include gc
 
+protected lemma dual [pα : preorder α] [pβ : preorder β]
+  {l : α → β} {u : β → α} (gc : galois_connection l u) :
+  @galois_connection (order_dual β) (order_dual α) _ _ u l :=
```

Comment:
Shouldn't `l` and `u` be composed with `to_dual`?

Reply:
I moved this lemma without touching the code. I mostly use `order_dual` for `@dual_lemma (order_dual α) _ _` proofs, so I don't care about `to_dual`. Feel free to add it if you want to. I guess, something like
```suggestion
  galois_connection (order_dual.to_dual ∘ u ∘ order_dual.of_dual)
    (order_dual.to_dual ∘ l ∘ order_dual.of_dual) :=
```
should work.

Reply:
Oh, I missed that this lemma was moved. Yeah, that's basically what I was thinking . You'll need to `import order.order_dual`, but that seems like a good idea to change to me.